connection/crossBorder.js

```diff
@@ -2,6 +2,7 @@ const Handshake = require('../packets/clientbound/handshake.js');
 const ProxyLoginStart = require('../packets/clientbound/proxyLoginStart.js');
 const remittanceLogin = require('../connection/handleLogin.js').remittance;
 const net = require('net')
+const log = require('loglevel')
```

Comment:
New logging library since it makes no sense to make our own logging library from scratch